src/main/java/net/bhardy/bizzo/billing/impl/PolicyBuilderImpl.java

```diff
@@ -36,6 +36,13 @@
  * @see net.bhardy.bizzo.billing.PolicyBuilder
  */
 public class PolicyBuilderImpl implements PolicyBuilder {
+
+    public int[] PROBLEM;
+
+    public PolicyBuilderImpl() {
+        int t= PROBLEM.length;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused "t" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1481)